test/EndToEnd.Tests/GivenFrameworkDependentApps.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Tools.Test.Utilities;
+using NuGet.ProjectModel;
+using NuGet.Versioning;
+using RestoreCommand = Microsoft.DotNet.Tools.Test.Utilities.RestoreCommand;
```

Comment:
There's a lot of these jank `using` aliases to make these tests not conflict with the same/similar concepts in the Microsoft.NET.TestFramework. An effort to merge Test.Utilities into it will need to be done post-merge.